test/spec/modules/appnexusBidAdapter_spec.js

```diff
@@ -1076,6 +1076,16 @@ describe('AppNexusAdapter', function () {
           'adUnitCode': 'code',
           'appnexus': {
             'buyerMemberId': 958
+          },
+          'meta': {
+            'dchain': {
+              'ver': '1.0',
+              'complete': 0,
+              'nodes': [{
+                'bsid': '958',
```

Comment:
I think this dchain would have two nodes, one for xandr as an ssp and one for the buyer seat id. The final node would be for Xandr alone, and the second to final node would have the bsid alone.

Reply:
Hi @patmmccann 

Just to clarify - is the 'xandr' node you're referring to represent the same node as the 'bidder' node that the dchain module appends later when the bid is processed?  Or is meant to represent something different for the xandr adserver system?

If our bidder creates two nodes now, first for the bsid and the second for the xandr name, eventually there will be a third node attached with the appnexus bidder name (attached by the module's workflow); is that the ideal chain that should be made?